README.md

```diff
@@ -132,50 +128,28 @@ Set up a new local CMS instance.
 > A local machine CMS will be empty. It will **not** have content from staging nor production. To have that, follow and adapt instructions to [copy a database](https://confluence.tacc.utexas.edu/x/W4DZDg).
 
 > **Note**
-> A local machine CMS does **not** include **nor** integrate with an instance of [Core Portal]. There are no reliable instructions to do either. **Help welcome.**
+> A local machine CMS does **not** include **nor** integrate with an instance of [Core Portal]. To attempt to do that, follow [How to Use a Custom Docker Compose File](https://github.com/TACC/Core-CMS/wiki/How-to-Use-a-Custom-Docker-Compose-File) and [Locally Develop CMS Portal Docs](https://github.com/TACC/Core-CMS/wiki/Locally-Develop-CMS---Portal---Docs). **Help welcome.**
 
 ## Update Project
 
-To update an existing CMS instance.
-
-### New Major [Core CMS] Version (or v3.12)
-
-Read [Upgrade Project] for developer instructions.
+Follow [Core CMS: Update Project](https://github.com/TACC/Core-CMS/blob/main/README.md#update-project) instructions.
 
-### New Branch (or Minor or Patch [Core CMS] Version)
+## Develop Project
 
-1. If CMS Docker files changed, rebuild Docker Containers:
-
-    ```sh
-    cd custom_project_dir
-    make stop
-    make build
-    make start
-    ```
-
-2. If static assets or database models changed, update the Django Application:
-
-    ```sh
-    docker exec -it core_cms /bin/bash
-    # That opens a command prompt within the container.
-        python manage.py migrate
-        python manage.py collectstatic --no-input
-        # If the project has no new/changed assets,
-        # then expect output of "0 static files […]"
-    ```
-
-## Run Multiple Projects
-
-Read [Multiple Projects](./docs/run-project.md#multiple-projects).
+Follow "via Core CMS Resources" section of [Core CMS: Develop Custom Project](https://github.com/TACC/Core-CMS/blob/main/docs/develop-custom-project.md#via-core-cms-resources).
 
 ## Debug Project
 
-Read [Debug Project](./docs/debug-project.md) for miscellaneous tips.
+Read [Core CMS: Debug Project](https://github.com/TACC/Core-CMS/blob/main/docs/debug-project.md).
 
 ## Build & Deploy Project
 
 Follow "Core-CMS-Resources" section of [How To Build & Deploy][Build & Deploy Project].
 
+## Run Multiple Projects
+
+Read [Run Multiple Projects](./docs/run-multiple-projects.md).
+
```

Comment:
The file's name and scope changed. And I moved this section to after all the sections that mimic those of Core-CMS's README.